### ISSUE TITLE
Inset doesn't seem to make sense.

### ISSUE THREAD

**Issue by kjsolo** [Fondesa/RecyclerViewDivider#29]
```kotlin
var insetBefore: Int = insetManager.itemInsetBefore(groupCount, groupIndex)
var insetAfter: Int = insetManager.itemInsetAfter(groupCount, groupIndex)
if (spanCount > 1 && (insetBefore > 0 || insetAfter > 0)) {
    insetBefore = 0
    insetAfter = 0
    Log.e(TAG, "the inset won't be applied with a span major than 1.")
}

...
// 1. insetBefore and insetAfter always 0 when spanCount > 2
// 2. insetBefore and insetAfter never used when spanCount == 1

if (orientation == RecyclerView.VERTICAL) {
    when {
        // LinearLayoutManager or GridLayoutManager with 1 column
        spanCount == 1 || spanSize == spanCount -> setRect(0, 0, 0, size)
        // first element in the group
        lineAccumulatedSpan == spanSize -> setRect(0, 0, halfSize + insetAfter, size)
        // last element in the group
        lineAccumulatedSpan == spanCount -> setRect(halfSize + insetBefore, 0, 0, size)
        // element in the middle
        else -> setRect(halfSize + insetBefore, 0, halfSize + insetAfter, size)
    }
}
```

**Comment by Fondesa** [comment]
You are right, the inset should be ignored in `getItemOffsets()` and used only in `onDraw()`.
Now the inset is correctly used in `onDraw` so the visual effect is correct.
The code related to insets in `getItemOffsets()` can be removed since, as you pointed out, it doesn't make any sense.

Since now the inset is working (look at the green box with inset-after set to `70`), I'm not releasing a fix for it, but I'm removing the useless code from `getItemOffsets()` for the next release.
Thanks.

<img src="https://user-images.githubusercontent.com/13958744/65831705-22ad7480-e2bd-11e9-8eaf-0946bb40d081.png " width="300">

**Comment by Fondesa** [comment]
Here we go: https://github.com/Fondesa/RecyclerViewDivider/pull/30

**Comment by Fondesa** [comment]
Released in **3.0.1**.
